pkg/http/api.go

```diff
@@ -698,6 +733,128 @@ func (a *api) getLockStoreWithRequestValidation(reqCtx *fasthttp.RequestCtx) (lo
 	return a.lockStores[storeName], storeName, nil
 }
 
+// Route:   "workflows/{workflowComponent}/{workflowType}/{instanceId}",
+// Workflow Component: Component specified in yaml (temporal, etc..)
+// Workflow Type: Name of the workflow to run (function name)
+// Instance ID: Identifier of the specific run
+func (a *api) onStartWorkflow(reqCtx *fasthttp.RequestCtx) {
+	startReq := wfs.StartRequest{}
+
+	wfType := reqCtx.UserValue(workflowType).(string)
+	if wfType == "" {
+		log.Debug("No workflow, or empty workflow was provided.")
+		return
+	}
+
+	component := reqCtx.UserValue(workflowComponent).(string)
+	if component == "" {
+		log.Debug("No component, or empty component was provided.")
+		return
+	}
+	if a.workFlows[component] == nil {
+		log.Debug("The provided component does not exist.")
+		return
+	}
+
+	err := json.Unmarshal(reqCtx.PostBody(), &startReq)
+	if err != nil {
+		log.Debug(err)
+	}
+
+	instance := reqCtx.UserValue(instanceId).(string)
+	if instance == "" {
+		log.Debug("No instance, or empty instance was provided.")
+		return
+	}
+
+	req := workflows.StartRequest{
+		Options:      startReq.Options,
+		WorkflowName: wfType,
+		Parameters:   startReq.Parameters,
+	}
+	req.WorkflowInfo.InstanceId = instance
+
+	resp, err := a.workFlows[component].Start(reqCtx, &req)
+	if err != nil {
+		log.Debug(err)
```

Comment:
If there's an error, we should `log.Error` it - also we should probably handle it here and not continue, right?